Sources/MapboxNavigation/MGLStyle.swift

```diff
@@ -87,4 +88,129 @@ extension MGLStyle {
             removeSource(source)
         }
     }
+
+    /**
+     Convenience method for adding a circle at a given coordinate.
+
+     Useful for debugging or visualizing data.
+     */
+    public func addDebugCircleLayer(identifier: String, coordinate: CLLocationCoordinate2D, color: UIColor = UIColor.purple) {
```

Comment:
Mention these new public methods (by fully-qualified name, like `MGLStyle.addDebugCircleLayer(identifier:coordinate:color:)` in the changelog, or make them private if they aren’t intended to be used outside the SDK.